### PR TITLE
APOLLO-28026: Update connectors service names

### PR DESCRIPTION
**Pull request by SemionPar**
Change connectors service names

### CODE REVIEW

**Review by acesar — APPROVED** [review]
LGTM - small comment.

File: survival_guide/2_planning.adoc [hunk 1]
```diff
@@ -359,8 +359,8 @@ lucidworks/ml-model-service:5.1.0
 lucidworks/ml-python-image:5.1.0
 lucidworks/pm-ui:5.1.0
 lucidworks/query-pipeline:5.1.1
-lucidworks/rest-service:5.1.1
-lucidworks/rpc-service:5.1.1
+lucidworks/connectors:5.1.1
```
[the remark this answers is not in the document]
Comment (luis-munoz) [reply survival_guide/2_planning.adoc:362]: lgtm, but I also think that the survival guide should be updated with the new versions

[the remark this answers is not in the document]
Comment (SemionPar) [reply survival_guide/2_planning.adoc:362]: Can we update all of the `lucidworks/` deps here to 5.4.0 then?

[the remark this answers is not in the document]
Comment (acesar) [reply survival_guide/2_planning.adoc:362]: let's revet the change here - and file a new ticket to upgrade this sections for 5.4.x, sine the required changes do not only concern connector.

[the remark this answers is not in the document]
Comment (SemionPar) [reply survival_guide/2_planning.adoc]: @acesar filed the ticket here: https://lucidworks.atlassian.net/browse/APOLLO-28955

**Review by luis-munoz — APPROVED** [review]
Lgtm

**Pull request merged**